src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -4027,6 +4032,28 @@ internal void CheckStack(string trace)
                     Debug.Assert(_stateObj._permitReplayStackTraceToDiffer || prev.Stack == trace, "The stack trace on subsequent replays should be the same");
                 }
             }
+
+            public int CurrentPacketIndex
```

Comment:
What's the significance of this method? Was this something you were using for debugging?